style.css

```diff
@@ -1,36 +1,860 @@
+    display: none;
+  }
+
+  .mob-nav {
+    display: none;
+  }
+
+  .hidden {
+    display: none;
+  }
+
+  .mainn-nav {
+    background-color: black;
+    height: 5vh;
+  }
+
+  .main-nav-ul {
+    display: flex;
+    flex-direction: row;
+    column-gap: 15px;
+    color: rgb(255, 255, 255);
+  }
+
+  .navigation {
+    display: flex;
+    flex-direction: row;
+    column-gap: 20%;
+  }
+
+  #logos {
+    margin-left: 15%;
+    padding-top: 8px;
+  }
+
+  .second-nav-ul {
+    display: flex;
+    flex-direction: row;
+    column-gap: 10px;
+    margin-left: 2%;
+    width: 100%;
+    padding-top: 1.5%;
+  }
+
+  .second-nav-ul li a {
+    color: rgb(77, 77, 77);
+  }
+
+  .second-nav-ul li a:hover {
+    color: orangered;
+  }
+
+  .navigation-logo {
+    width: 20%;
+    margin-left: 10%;
+  }
+
+  .navigation-logos {
+    max-width: max-content;
+    height: auto;
+    background-color: red;
+  }
+
+  #hide {
+    display: none;
+  }
+
+  #cc-btn {
+    border: solid orangered;
+    color: orangered;
+  }
+
+  .main-section {
+    background: url(Img/desktop-about-bg_img.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 100vh;
+  }
+
+  .hello {
+    width: 80%;
+  }
+
+  .main-text {
+    display: flex;
+    flex-direction: column;
+    font-family: "Lato", sans-serif;
+    margin-left: 14%;
+    padding-top: 5%;
+    color: orangered;
+  }
+
+  .top {
+    font-size: 30px;
+    padding-bottom: 8%;
+  }
+
+  .middle {
+    font-size: 70px;
+    width: 100%;
+    font-family: "Lato", sans-serif;
+    font-weight: bolder;
+    margin-top: -70px;
+  }
+
+  .lower {
+    font-size: 16px;
+    margin-top: 5%;
+    width: 60%;
+    color: black;
+    background-color: rgb(235, 235, 235);
+    border: solid white;
+  }
+
+  .date {
+    margin-top: 5%;
+    color: black;
+  }
+
+  .date h4 {
+    color: black;
+    font-weight: bolder;
+    font-family: "Lato", sans-serif;
+    font-size: 25px;
+  }
+
+  .contact {
+    margin-left: 30%;
+    margin-top: 7%;
+    font-weight: bolder;
+    font-size: 20px;
+  }
+
+  .section {
+    background: url(Img/mainbg.webp);
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: max-content;
+    display: flex;
+    flex-direction: column;
+    overflow-y: hidden;
+    overflow-x: hidden;
+  }
+
+  .section-container {
+    display: flex;
+    width: 80%;
+    padding-top: 4%;
+    margin-left: 10%;
+    margin-right: 25%;
+    margin-top: 5%;
+  }
+
+  .program-box {
+    display: flex;
+    flex-direction: column;
+    margin-left: 10px;
+    background: rgba(121, 121, 121, 0.3);
+    width: 100%;
+    font-family: "Lato", sans-serif;
+  }
+
+  .program-icon {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .program-icon h3 {
+    font-size: 20px;
+    color: orangered;
+  }
+
+  .program-icon img {
+    width: 35%;
+  }
+
+  .program-box p {
+    color: white;
+    padding: 3%;
+    font-size: 15px;
+  }
```

Comment:
- [ ] I have noticed that the program-box paragraph, over crosses the borders of the box. This decreases the user interface of the application and therefore not user-friendly. Kindly consider fixing this so that your design matches the original design ☺️ 

Your design at 713px
![image](https://github.com/Zel-hub7/Capstone-project1/assets/99394374/94672375-ac8e-4027-bfa7-16d4c3103b44)

Original design 😎 
![image](https://github.com/Zel-hub7/Capstone-project1/assets/99394374/ec688123-791c-4c42-958d-e4227d8d7c8f)